### PR TITLE
Add description for license URL

### PR DESCRIPTION
**Pull request by ThomasJanotta**
Add description for license URL

### CODE REVIEW

File: docs/manual/file-manager/meta-data.en.md [hunk 1]
```diff
@@ -33,3 +34,40 @@ Contao supports the following metadata:
     </figure>
 </div>
 ```
+
+**License URL**
+{{< version "4.13" >}}
+
+The license URL is displayed on the corresponding page on which the image is embedded in [JSON-LD format](https://en.wikipedia.org/wiki/JSON-LD") as [SCHEMA-ImageObject entry](https://schema.org/ImageObject "SCHEMA-ImageObject") included to comply with licensing notices.
```
Comment (fritzmg) [inline docs/manual/file-manager/meta-data.en.md]:
```suggestion
On the page on which the image is embedded the license URL is used in the [SCHEMA-ImageObject](https://schema.org/ImageObject "SCHEMA-ImageObject") of the [JSON-LD](https://en.wikipedia.org/wiki/JSON-LD") to comply with licensing notices.
```

File: docs/manual/file-manager/meta-data.en.md [hunk 2]
```diff
@@ -33,3 +34,40 @@ Contao supports the following metadata:
...
+                "name": "Contao CMS"
+            }
+        ]
+    },
+    ...
+]
+</script>
+```
+
+To specify the license URL under the image on the page, you can add the following code to the image.html5 template:
```
Comment (fritzmg) [inline docs/manual/file-manager/meta-data.en.md]:
```suggestion
To display the license URL under the image on the page you can add the following code to the `image.html5` template:
```

File: docs/manual/file-manager/meta-data.en.md [hunk 3]
```diff
@@ -33,3 +34,40 @@ Contao supports the following metadata:
...
+
+```php
+<?php if($this->license): ?><p class="ce_image__license" ><?= $this->license ?></p><?php endif; ?>
```
Comment (fritzmg) [inline docs/manual/file-manager/meta-data.en.md]:
```suggestion
<?php if ($this->license): ?><p class="ce_image__license" ><?= $this->license ?></p><?php endif; ?>
```

**Review by fritzmg — CHANGES_REQUESTED** [review]

**Comment by ThomasJanotta** [comment]
OK

**Review by fritzmg — APPROVED** [review]

**Review by netzarbeiter — APPROVED** [review]
Thank you, Thomas

**Pull request merged**